_posts/2021-11-30-foreman-community-newsletter-november-2021.md

```diff
@@ -0,0 +1,120 @@
+---
+
+In somewhat of a contribution special, take a look at some easy ways you can help the Foreman community today.
+
+<!--more-->
+
+## How can I make easy contributions to Foreman?
+
+People might assume that contributing to an open source project means contributing code.
+While PRs are always welcome over on [theforeman's GitHub repo](https://github.com/theforeman/), this is far from true.
+There are many ways to contribute to Foreman.
+Here are some quick ways you can help Foreman today.
+
+At the moment, we are preparing for the releases of Foreman 3.1 and Katello 4.3.
+One way you could make a big contribution to the project is to install the release candidates and try out some of your regular workflows, or explore the new features, and [let us know if everything looks OK](https://community.theforeman.org/).
+This contributes to the stability of the future release.
+
+## Evaluate Foreman 3.1 RC 2
+
+As well as many bug fixes, with the Foreman 3.1 release candidate, you can evaluate the following new features:
+
+* Enhancements to the new host details page
+* Descriptions for all provisioning templates
+* A new purge:puppet rake task
+* Cron job added to help manage Audit cleanups
+* Provisioning snippets support Puppet 7
+
+For more information about how to install this release candidate, check out the [Foreman 3.1 release candidate announcement](https://community.theforeman.org/t/foreman-3-1-rc2-is-ready-for-testing/26368/).
+
+## Evaluate Katello 4.3 RC 1
+
+As well as bug fixes, with the Katello 4.3 release candidate, you can evaluate the following new features:
+
+* A complete update to the Content View web UI and associated workflows
+* Some OSTree content support
+* Oracle ULN content support
+* Inter Server Sync (ISS) available for connected workflows
+* ....And much more.
+
+For information about how to install this release candidate, check out the [Katello 4.3 release candidate announcement](https://community.theforeman.org/t/katello-4-3-rc1-is-now-ready-for-testing/26349) on our Foreman Community Discourse.
+
+## Tell us what you think about proposed changes
+
+Another quick and easy way to help the future of Foreman is to share your opinion on our [Foreman Community Discourse](https://community.theforeman.org/). Here are some hot topics from the last few weeks.
+
+## Centos 8 sunsets soon; what next?
+
+At the moment, we are preparing for our next steps after Centos 8 sunsets at the end of this year.
+
+[ehelms](https://community.theforeman.org/u/ehelms) lays out the options he sees for what to do about building of RPMs and testing of RPMs for future Foreman releases.
+
+He outlines his thoughts on two possible options:
+
+* Option 1: Build against RHEL
+* Option 2: Build against CentOS Stream
+
+Please let him know what you think over on his post: [RFC - Enterprise Linux 8 and 9 Support](https://community.theforeman.org/t/rfc-enterprise-linux-8-and-9-support/26282).
+
+## How do you use organization/locations in  Foreman?
```

Comment:
Double space.

Reply:
I am the queen of double spaces. @spetrosi can testify to that ;)